addons/analytic/static/src/components/analytic_distribution/analytic_distribution.xml

```diff
@@ -2,34 +2,27 @@
 <templates xml:space="preserve">
 
     <t t-name="analytic_distribution" owl="1">
-        <div class="analytic_distribution" t-ref="analyticDistribution" t-on-keydown="onWidgetKeydown">
-            <t t-call="analytic_distribution.tags"/>
-            <t t-if="!props.readonly">
-                <div class="o_input o_input_dropdown hidden-focus" tabindex="0" t-ref="mainElement" t-on-focus="onMainElementFocus">
-                    <a role="button" class="o_dropdown_button" draggable="false" />
-                    <t t-call="analytic_distribution.popup"/>
-                </div>
-            </t>
-        </div>
+        <t t-call="analytic_distribution.tags" t-ref="analyticDistribution" t-on-keydown="onWidgetKeydown"/>
     </t>
 
     <t t-name="analytic_distribution_form" owl="1">
-        <div class="analytic_distribution" t-ref="analyticDistribution" t-on-keydown="onWidgetKeydown">
-
-            <t t-call="analytic_distribution.tags"/>
-            <t t-if="!props.readonly">
-                <div class="o_input o_input_dropdown hidden-focus" tabindex="0" t-ref="mainElement" t-on-focus="onMainElementFocus" t-on-click="onMainElementFocus">
```

Comment:
this `t-on-click` is different for the form view

Reply:
Correct. I fixed this in my latest revision, showing only all edit functionality when we're not in readonly mode.